src/styles/catalog.scss

```diff
@@ -0,0 +1,68 @@
+.catalog {
+  margin: auto 0;
+  padding: 0 50px;
+  background-color: #fff;
+  box-shadow: 0 2 4 0 #000 5%;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.logo__image {
+  height: 40px;
+  width: 40px;
+}
+
+.logo {
```

Comment:
Why do you have styles for `logo` inside catalog.scss if logo is a separate BEM block? Create a separate logo.scss file